lib/stdlib/src/sys.erl

```diff
@@ -582,9 +600,9 @@ trim(N, LogData) ->
 %% Debug structure manipulating functions
 %%-----------------------------------------------------------------
 install_debug(Item, Data, Debug) ->
-    case get_debug2(Item, Debug, undefined) of
-	undefined -> [{Item, Data} | Debug];
-	_ -> Debug
+    case lists:keysearch(Item, 1, Debug) of
```

Comment:
Sorry for being picky - I'm probably missing something... Why did you change these three lines? Didn't get_debug2/3 do the job?

Reply:
actually `get_debug/2` generates above bug! It yields `undefined` for a function with state `undefined` and we don't want to add it again.

Reply:
Ah, yes of course! That's the bugfix. Thanks for clarifying!